docs/iris/src/developers_guide/release.rst

```diff
@@ -0,0 +1,72 @@
+Releases
+********
+
+A release of Iris is a tag on github in the SciTools organisation master.
+
+
+Release Branch
+==============
+
+Once the features intended for the release are on master, a release branch should be created, in the SciTools/iris repository.  This will have the name:
+
+    :literal:`{major release number}.x`
+
+for example:
+
+    :literal:`v1.9.x`
+
+This branch shall be used to finalise the release details in preparation for the release candidate.
+
+Release Candidate
+=================
+
+Prior to a release, a release candidate tag shall be created, marked as a pre-release in github, with a tag ending with :literal:`rc` followed by a number, e.g.:
```

Comment:
Perhaps we can use the word MAY. That indicates that it isn't *always* necessary.